src/interfaces/IERC3009.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.23;
+
+interface IERC3009 {
+    /**
+     * @notice Emitted when an authorization has been used.
+     * @param  authorizer    Authorizer's address
+     * @param  nonce         Nonce of the used authorization
+     */
```

Comment:
Alignment on natspec for all these functions.

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/7/commits/287b68343734e04236e9d631ee9a4d0e8db50a21